openvpn.c

```diff
@@ -64,13 +64,14 @@ TerminateOpenVPN(connection_t *c);
 
 const TCHAR *cfgProp = _T("conn");
 
-#define FLAG_CR_TYPE_SCRV1 0x1    /* static challenege */
-#define FLAG_CR_TYPE_CRV1  0x2    /* dynamic challenege */
-#define FLAG_CR_ECHO       0x4    /* echo the response */
-#define FLAG_CR_RESPONSE   0x8    /* response needed */
-#define FLAG_PASS_TOKEN    0x10   /* PKCS11 token password needed */
-#define FLAG_STRING_PKCS11 0x20   /* PKCS11 id needed */
-#define FLAG_PASS_PKEY     0x40   /* Private key password needed */
+#define FLAG_CR_TYPE_SCRV1  0x1    /* static challenege */
+#define FLAG_CR_TYPE_CRV1   0x2    /* dynamic challenege */
+#define FLAG_CR_TYPE_CRTEXT 0x80   /* crtext */
```

Comment:
Please move this to the end of the list, below FLAG_PASS_KEY. Keeping the values in order makes it easier to find what the next unused value is.

Reply:
I was unsure should it be ordered by name or value. Will move to the end.